docs/guides/expectations/features_custom_expectations/how_to_add_example_cases_for_an_expectation.md

```diff
@@ -0,0 +1,152 @@
+
+```python file=../../../../tests/integration/docusaurus/expectations/creating_custom_expectations/expect_column_max_to_be_between_custom.py#L46
+```
+
+:::note
+When you define data in your examples, we will mostly guess the type of the columns. 
+Sometimes you need to specify the precise type of the columns for each backend. Then you use the `schema` attribute (on the same level as `data` and `tests` in the dictionary):
+
+```console
+"schemas": {
+  "spark": {
+    "x": "IntegerType",
+  },
+  "sqlite": {
+    "x": "INTEGER",
+  },
+```
+:::
+
+#### 3. Defining our tests
+
+In our example, `tests` will be a list containing dictionaries defining each test. 
+
+You will need to:
+1. Title your tests (`title`);
+2. Define the input for your tests (`in`);
+3. Decide how precisely you want to test the output of your tests (`exact_match_out`);
+4. Define the expected output for your tests (`out`).
+
+If you are interested in contributing your Custom Expectation back to Great Expectations, you will also need to decide if you want these tests publically displayed to demonstrate the functionality of your Custom Expectation (`include_in_gallery`).
+
+```python file=../../../../tests/integration/docusaurus/expectations/creating_custom_expectations/expect_column_max_to_be_between_custom.py#L44-L76
+```
+
+#### 4. Verifying our tests
+
+If you now run your file, `print_diagnostic_checklist` will attempt to execute these example cases.
+
+If the tests are correctly defined, and the rest of the logic in your Custom Expectation is already complete,
+you will see the following in your Diagnostic Checklist:
+
+```console
+✔ Has at least one positive and negative example case, and all test cases pass
+```
+
+<div style={{"text-align":"center"}}>
+<p style={{"color":"#8784FF","font-size":"1.4em"}}><b>
+Congratulations!<br/>&#127881; You've successfully created example cases & tests for a Custom Expectation! &#127881;
+</b></p>
+</div>
+
+#### 5. Contribution (Optional)
+
+This guide will leave you with test coverage sufficient for [contribution](../contributing/how_to_contribute_a_new_expectation_to_great_expectations.md) back to Great Expecations at an Experimental level;
+if you're interested in having your contribution accepted at a Beta level, these tests will need to pass for all supported backends (Pandas, Spark, & SQLAlchemy).
+
+For full acceptance into the Great Expectations codebase at a Production level, we require a more robust test suite. 
+If you believe your Custom Expectation is otherwise ready for contrbution at a Production level, please submit a [Pull Request](https://github.com/great-expectations/great_expectations/pull-requests), and we will work with you to ensure adequate testing.
+
+:::note
+For more information on our code standards and contribution, see our guide on [Levels of Maturity](../../../contributing/contributing_maturity.md#contributing-expectations) for Expectations
+:::
```

Comment:
```suggestion
:::note
For more information on our code standards and contribution, see our guide on [Levels of Maturity](../../../contributing/contributing_maturity.md#contributing-expectations) for Expectations.

To view the full script used in this page, see it on GitHub:
- [expect_column_max_to_be_between_custom.py](https://github.com/great-expectations/great_expectations/blob/hackathon-docs/tests/integration/docusaurus/expectations/creating_custom_expectations/expect_column_max_to_be_between_custom.py)
:::
```
Good job on using file and reference numbers for the code examples!  Our documentation standards are such that at the end of a document that does this (as all documents with code examples ideally should) we also want to link to the full code so the user can reference it in its entirety if they need to.  You may need to adjust the link that is used in this suggestion: it is pointing to the file in the hackathon-docs branch, since that file doesn't exist at that location in other branches currently.